android/java/org/chromium/chrome/browser/app/domain/PortfolioModel.java

```diff
@@ -159,17 +160,17 @@ public void fetchAssetsByType(TokenUtils.TokenType type, NetworkInfo selectedNet
                                 mPortfolioHelper.fetchAssetsAndDetails(type, callback);
                             });
                         } else {
-                            mKeyringService.getKeyringInfo(
-                                    AssetUtils.getKeyring(
-                                            selectedNetwork.coin, selectedNetwork.chainId),
-                                    keyringInfo -> {
-                                        mPortfolioHelper =
-                                                new PortfolioHelper(braveWalletBaseActivity,
-                                                        mAllNetworkInfos, keyringInfo.accountInfos);
-                                        mPortfolioHelper.setSelectedNetworks(
-                                                Arrays.asList(selectedNetwork));
-                                        mPortfolioHelper.fetchAssetsAndDetails(type, callback);
-                                    });
+                            mKeyringService.getAllAccounts(allAccounts -> {
+                                AccountInfo[] filteredAccounts =
+                                        AssetUtils.filterAccountsByNetwork(allAccounts.accounts,
+                                                selectedNetwork.coin, selectedNetwork.chainId);
+
+                                mPortfolioHelper = new PortfolioHelper(braveWalletBaseActivity,
+                                        mAllNetworkInfos, filteredAccounts);
+                                mPortfolioHelper.setSelectedNetworks(
+                                        Arrays.asList(selectedNetwork));
+                                mPortfolioHelper.fetchAssetsAndDetails(type, callback);
+                            });
                         }
```

Comment:
`getKeyringInfo` is deprecated and to be removed.
mKeyringService.getAllAccounts() is the only source of accounts list now